js/background.js

```diff
@@ -166,7 +170,33 @@ chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
 });
 
 chrome.tabs.onActivated.addListener(function(activeInfo){
-  capturePage(activeInfo.tabId);
+  let tabId = activeInfo.tabId;
+
+  chrome.tabs.captureVisibleTab({quality: 1}, function(screenshotUrl) {
+    var lastError = chrome.runtime.lastError;
+    if (!lastError) {
+      chrome.tabs.query({active: true}, function(tabs){
+        let currentTabId = tabs[0].id;
+        if(currentTabId == tabId)
+          saveImage(tabId, screenshotUrl);
+      });
+    }
+  });
+
+  //update activate time
+
+  getTabDetails(tabId, (tabDetails) => {
+    if (!tabDetails)
+      tabDetails = {};
+    let time = +new Date();
+    console.log("update active time : tabId:, time :", tabId, time)
```

Comment:
Please make sure any `console.log` to be `console.debug` to prevent logging at customer side